main.go

```diff
@@ -78,9 +78,8 @@ func main() {
 	repository := customers.NewJsonFileRepository(config.CustomersFile)
 	sqlDB := sqlx.MustConnect("sqlite3", config.DBConnectionString)
 	patientRepository := patients.NewSQLitePatientRepository(patients.Factory{}, sqlDB)
-	//patientRepository := patients.NewMemoryPatientRepository(patients.Factory{})
-	customers, _ := repository.All()
 	if config.LoadTestPatients {
+		customers, _ := repository.All()
```

Comment:
I'm aware this is a demo project. But even then we should probably handle the error right?